README.md

```diff
@@ -8,7 +8,7 @@ specified number of times until the example succeeds.
 
 | Rspec Version | Rspec-Retry Version |
 |---------------|---------------------|
-| 3.3-3.7       | 0.5.6               |
+| > 3.3         | 0.5.7               |
```

Comment:
Fwiw I'd rather keep the readme bound to the max of what's in the lockfile.  If we want to add a `> 3.7 | untested but try 0.5.7` that'd be a compromise.